### PR TITLE
Create unit tests for Partner link component

### PR DESCRIPTION
**Pull request by Rominos7**
### Description
THis PR adds:
1) Configuration for jest in order to use SVG icons by using [jest-transformer-svg](https://www.npmjs.com/package/jest-transformer-svg) lib
2) Adding expample unit tests for Partner link component

### How does it looks
Screenshot 2024-01-14 131107

### CODE REVIEW

**Review by ValDekh — APPROVED** [review]

**Review by malyna2 — APPROVED** [review]

File: jest.config.ts [hunk 1]
```diff
@@ -29,13 +29,16 @@ const jestConfig: JestConfigWithTsJest = {
     },
     verbose: true,
     collectCoverageFrom: [
-        "src/**/*.{ts, tsx, js, jsx}"
+        "src/**/*.{ts,js}" // temp disable coverage collection for tsx, jsx util type errors will be fixed
```
Comment (Oleh-Kril) [inline jest.config.ts:32]: maybe until ?

File: package.json [hunk 2]
```diff
@@ -124,6 +124,7 @@
     "@babel/plugin-proposal-private-property-in-object": "^7.21.11",
     "@babel/preset-env": "^7.23.3",
     "@babel/preset-react": "^7.23.3",
+    "@nestjs/testing": "^10.3.0",
```
Comment (Oleh-Kril) [inline package.json:127]: why do we need nestjs in react project ?

**Review by Oleh-Kril — COMMENTED** [review]
Please fix small comments so we can merge this pull request

File: jest.config.ts (hunk 1 above)
Comment (Rominos7) [reply jest.config.ts]: Done

File: package.json (hunk 2 above)
Comment (Rominos7) [reply package.json]: Done

**Review by Oleh-Kril — APPROVED** [review]

**Pull request merged**